src/main/java/org/jvnet/hudson/test/TestCrumbIssuer.java

```diff
@@ -5,39 +5,39 @@
  */
 package org.jvnet.hudson.test;
 
-import javax.servlet.ServletRequest;
-
-import net.sf.json.JSONObject;
-
-import org.kohsuke.stapler.StaplerRequest;
-
 import hudson.Extension;
 import hudson.model.ModelObject;
 import hudson.security.csrf.CrumbIssuer;
 import hudson.security.csrf.CrumbIssuerDescriptor;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.StaplerRequest;
+import javax.servlet.ServletRequest;
```

Comment:
Please revert unnecessary whitespace changes. Yes, the formatting is odd but commit history spoiled with irrelevant changes is worse.

Reply:
Well actually, since it is in separate commit, I am fine with it.